engine/loader.asm

```diff
@@ -235,9 +237,16 @@ SetFrontBufBackground@
             jsr         System_SetPaletteAuto
             lda         #0
             jsr         Img_FadeIn
-            * Allocate 8k block to store Level and Object code, and map it to $6000
-            lda         #VH_LVLOBJCODE
+            * Allocate 8k blocks to store Level and Object code, and map first to $6000
+            lda         #VH_LVLOBJCODEX
+!           pshs        a
```

Comment:
Register A is not modified by MemMgr_AllocateBlock, so you don't need to push and pull it here

Reply:
Nice!

Reply:
Fixed